go/vt/tabletmanager/state_change.go

```diff
@@ -272,19 +272,19 @@ func (agent *ActionAgent) changeCallback(ctx context.Context, oldTablet, newTabl
 		}
 	}
 
-	// update stream needs to be started or stopped too
+	// UpdateStream needs to be started or stopped too.
```

Comment:
I meant yes, they are unrelated. They're tiny though, so I'll leave them in there.